api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -137,6 +142,18 @@ public static void assertSerializedAndLoadedMetadata(Table expected, Table actua
     Assert.assertEquals("History must match", expected.history(), actual.history());
   }
 
+  public static void assertSerializedMetricsReporter(
+      MetricsReporter expected, MetricsReporter actual) {
+    Assert.assertNotNull("metrics reporter from serializableTable should not be null", actual);
+    Assert.assertTrue(
```

Comment:
this can be simplified to `Assertions.assertThat(actual).isNotNull().isInstanceOf(...)` and for the properties: `Assertions.assertThat(actual.properties).isEqualTo(...)`

Reply:
done, thanks